tiatoolbox/utils/misc.py

```diff
@@ -335,6 +337,59 @@ def contrast_enhancer(img, low_p=2, high_p=98):
     return np.uint8(img_out)
 
 
+def read_point_annotations(input_table):
+    """Read annotations as pandas DataFrame.
```

Comment:
I suggest the `out_table` should have three column in all cases, no matter even if the input has only 2 columns. This way, the `PointsPatchExtractor` can be written more consistently regarding the point class output.

Reply:
Added in this commit https://github.com/TIA-Lab/tiatoolbox/commit/478ec912426334b82d74e5a8e4c01494f62779ac